privacy.html

```diff
@@ -0,0 +1,448 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Vite App</title>
+    <link rel="stylesheet" href="dist/output.css" />
+    <link href="https://fonts.googleapis.com/css2?family=Outfit:wght@100..900&display=swap" rel="stylesheet" />
+
+    <script>
+        
+        const loadComponents = (id, url) => {
+            return fetch(url)
+                .then((response) => response.text())
+                .then((html) => {
+                    document.getElementById(id).innerHTML = html;
+                })
+        };
+
+        window.onload = () => {
+            loadComponents("footer", "src/sections/footer.html");
+        };
+    </script>
+</head>
+
+<body class="text-[#00672d]">
```

Comment:
can you use the template of the `index.html` and then put the privacy section in the `src/sections` folder too?